src/openrct2/management/Finance.cpp

```diff
@@ -196,6 +196,14 @@ void FinanceResetHistory()
         gWeeklyProfitHistory[i] = MONEY64_UNDEFINED;
         gParkValueHistory[i] = MONEY64_UNDEFINED;
     }
+
+    for (uint32_t i = 0; i < EXPENDITURE_TABLE_MONTH_COUNT; ++i)
+    {
+        for (uint32_t j = 0; j < static_cast<int32_t>(ExpenditureType::Count); ++j)
+        {
+            gExpenditureTable[i][j] = 0;
+        }
+    }
```

Comment:
You can get rid of the array dimensions here by either using a range based loops, or by using sizeof/std::size. You can even zero-initialize the entire array in a single memset or std::fill call.